dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -833,4 +833,6 @@ private Constants() {
     public static final int USER_PASSWORD_MIN_LENGTH = 2;
 
     public static final String FUNCTION_START_WITH = "$";
+
+    public static final Integer DEFAULT_TENANT_ID = 1;
```

Comment:
This is a tenant id or user id?

Reply:
the default tenant id, which is stored when the database is initialized.

Reply:
Ok, but I don't see we have inserted a default tenant into the database.

Reply:
Sorry, after I confirm again, the 'DEFAULT_TENANT_ID' constant here is the default queue ID, I will change the variable name.